### PR TITLE
Update `wholeBody_ct_segmentation`

### PR DESCRIPTION
**Pull request by KumoLiu**
### Description
While testing the lazy-resampling benchmark, I find that maybe there's room for improvement in the pre-processing. 
In this PR, I mainly update several places below:

- Remove `NormalizeIntensityd` since zeros aren't the background in the CT
- Move `ScaleIntensityd` in front of the `CropForeground` and change the value to 0-1 since we can only pad zero now in the lazy mode.
- Remove one-hot in the `AsDiscreted` which may save memory
- Update the metric in the training part from Accuracy to Dice
- Using the original label in the validation due to this [issue](https://github.com/Project-MONAI/model-zoo/issues/181#issuecomment-1510626663)

Note: the config was only tested on low resolution

cc @tangy5 @wyli 

### Status
**Work in progress**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by tangy5** [comment]
Thanks for the updates, looking forward to seeing the complete updates.

**Comment by Nic-Ma** [comment]
Hi @KumoLiu ,

Any update here?

Thanks.

**Comment by KumoLiu** [comment]
Hi @Nic-Ma, I don't have any update for this PR so far. The initial goal of this PR was to document what I thought could be improved in this model when I measured lazy resampling. However, because the data contained within this model zoo is actually high resolution, I was testing it on low resolution. So I'm just putting this PR here as a reference. If you think we need to keep updating it, I may need to redo some high-resolution experiments.
Thanks!
